src/trino/btrim_impl.rs

```diff
@@ -0,0 +1,143 @@
+// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Why these files are added?

Reply:
Probably a merge mistake--should I remove those changes?